### DISCUSSION TITLE
Inclusion of Godot Editor to F-Droid

### DISCUSSION THREAD

**Discussion by ClockGen** [godotengine/godot-proposals#6224] [General Discussions]
Hello, I'd like to include godot editor to F-Droid. F-Droid is an installable catalogue and repository of free and open source android apps, all of which are built from source.

There's several games using godot engine in F-Droid already, however, not the android port of godot editor itself.

As per inclusion policy, I need to notify the developers.

Rocket.chat thread (continuing discussion from there): https://chat.godotengine.org/channel/general/thread/4Rf5wENogD24HnRs9?jump=5hrvoM3wm68p664dy

F-Droid merge request: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/12540

**Comment by Calinou** [comment]
This is good to do on my end :slightly_smiling_face:
